spaces/domains.json

```
@@ -217,6 +217,7 @@
   "vote.moonpirate.finance": "moonpirate.eth",
   "vote.mai.finance" : "qidao.eth",
   "dao.modefi.io" : "mod.eth",
+  "vote.conqueruranus.com":"conqueruranus.eth",
```

Please correct the indentation of the file.
```suggestion
  "vote.conqueruranus.com": "conqueruranus.eth",
```